rust_src/src/keymap.rs

```diff
@@ -77,7 +77,7 @@ pub fn lookup_key(keymap: LispObject, key: LispObject, accept_default: LispObjec
         });
         idx += 1;
 
-        if c.is_cons() && lucid_event_type_list_p(c) {
+        if c.is_cons() && lucid_event_type_list_p(Option::<LispCons>::from(c)) {
```

Comment:
Isn't this `c.as_cons()`?

Reply:
well, technically I suppose it's c.as_cons_or_error()